src/openzeppelin/security/initializable.cairo

```diff
@@ -1,7 +1,7 @@
 #[contract]
 mod Initializable {
     struct Storage {
-        initialized: bool,
+        initialized: bool, 
```

Comment:
Is that change something with want. Trailing whitespaces are usually a big no.

Reply:
Good catch. When you include a trailing comma on a single variable, `cairo-format` automatically inserts a trailing space. Will fix